include/arch/arm/aarch64/timer.h

```diff
@@ -20,7 +20,8 @@ extern "C" {
 #define ARM_ARCH_TIMER_PRIO	ARM_TIMER_VIRTUAL_PRIO
 #define ARM_ARCH_TIMER_FLAGS	ARM_TIMER_VIRTUAL_FLAGS
 
-#define CNTV_CTL_ENABLE		((1) << 0)
+#define CNTV_CTL_ENABLE		BIT(0)
+#define CNTV_CTL_IT_MASK	BIT(1)
```

Comment:
If you want to keep the `_MASK` suffix please fix `CNTV_CTL_ENABLE` to keep it consistent. Otherwise just remove it.

Reply:
You mean tab aligned? In VIM, the two lines are aligned with linux coding style, not sure why github shows difference.

Reply:
Nope, I mean that you use the `_MASK` suffix for `CNTV_CTL_IT_MASK` but `CNTV_CTL_ENABLE` has no suffix at all. So either you use just `CNTV_CTL_IT` or you fix `CNTV_CTL_ENABLE` to `CNTV_CTL_ENABLE_MASK` for the sake of consistency.